### DISCUSSION TITLE
How do you stop bridging/overhangs, starting in thin air?

### DISCUSSION THREAD

**Discussion by Aurasphere-au** [OrcaSlicer/OrcaSlicer#6851] [Q&A]
I have tried so many diff settings and no matter what, I have critical bridging starting in thin air?

Bridging doing this
![image](https://github.com/user-attachments/assets/3ce374b5-36ab-4432-8022-30e85bcbab30)

Overhangs doing this
I have tried all the combos of infill first/inner outer/outer inner etc as well

![image](https://github.com/user-attachments/assets/30a6de85-cce9-498b-97c4-f4d75a569cba)

Then intermittent bits of diff line type on the same run/layer
![image](https://github.com/user-attachments/assets/13849496-949c-436c-a4d2-a92849ecf750)

**Comment by OrKaraz** [comment]
Hi,

currently the same problem. The only solution find at the moment is to set "nearest position" .
![Orca setting](https://github.com/user-attachments/assets/62a4314c-b96c-4c40-b224-c004e8b49e15)

I hope dev will get an eye on it to fix it.
